README.md

```diff
@@ -27,6 +27,8 @@ Kick off your project with this Startup Guide. This starter will help to install
 
     
 2.  **Test drive**
+
+
     This  describes how to create a new Flutter app from templates, run it, and experience “hot reload” after you make changes to the app.
```

Comment:
@arps18 Please keep in mind that we are not asking users to install and test flutter. Please remove this line.

Reply:
The entire line or just format the context?

Reply:
I would suggest removing the whole Test Drive content, as such, it is of no use.